hudi-client/src/main/java/org/apache/hudi/index/bloom/BucketizedBloomCheckPartitioner.java

```diff
@@ -144,7 +144,12 @@ public int numPartitions() {
   @Override
   public int getPartition(Object key) {
     final Pair<String, String> parts = (Pair<String, String>) key;
-    final long hashOfKey = Hashing.md5().hashString(parts.getRight(), StandardCharsets.UTF_8).asLong();
+    long hashOfKey = 0L;
```

Comment:
can we move this try-catch block to a method for reuse?  Also let's have that throw a `HoodieRuntimeException` instead of logging the error.. we cannot really make reasonable progress otherwise..  ? 

Should we change from MD5 to something else? We are not doing anything secure here.. just want something that will give good uniform hashing

Reply:
replaced with SHA-256 - but got'ta test it out on long strings to see how it works.

Reply:
CI tests passed with SHA-256.

Reply:
Reverting back to MD5